src/api/likes.ts

```diff
@@ -0,0 +1,62 @@
+import { db } from '@/api/firebaseApp'
+import { doc, getDoc, updateDoc, setDoc, deleteDoc } from 'firebase/firestore'
+
+export const fetchLikes = async (playlistId: string) => {
+  const docRef = doc(db, 'Likes', playlistId)
+  const docSnap = await getDoc(docRef)
+
+  if (docSnap.exists()) {
+    const data = docSnap.data()
+    return {
+      likeCount: Number(data.likeCount) || 0,
+      userLikes: data.userLikes || {}
+    }
+  } else {
+    return { likeCount: 0, userLikes: {} }
+  }
```

Comment:
P4;

참고차 말씀드리자면, 이런 식으로도 표현 가능합니다.

early- return이라 하며, if문에 해당하면 이미 return되어 함수가 종료되므로, else문이 없어도 되기 때문에 사용하는 패턴인데요.

가독성이 좋아 선호하는 사람들도 있지만, if-else 형태로 사용하는 게 잘못된 것은 아닙니다 (개인 선호에 따름)
```suggestion
  if (docSnap.exists()) {
    const data = docSnap.data()
    return {
      likeCount: Number(data.likeCount) || 0,
      userLikes: data.userLikes || {}
  }
  return { likeCount: 0, userLikes: {} }
```

Reply:
아하 ㅎㅎ 감사합니다!